sites/all/themes/orwell/sass/bootstrap/_colors.scss

```diff
@@ -8,14 +8,17 @@ $color__font-gray--light: #b4b4b4;
 $color__red: #d8133f;
 $color__red--light: #eca7b1;
 $color__white: #ffffff;
+
 $color__gray--lighter: #f0f0f0;
 $color__gray--light: #dcdcdc;
-$color__gray: gray;
+$color__gray: gray; // #808080
```

Comment:
Why not just set #808080 as color?

Reply:
Because Slott did it that way, and I'm just a lowly backend coder.

(will probably fix that up soon)